libminifi/src/core/ProcessGroup.cpp

```diff
@@ -412,6 +414,18 @@ void ProcessGroup::drainConnections() {
   }
 }
 
+std::size_t ProcessGroup::getTotalFlowFileCount() const {
+  std::size_t sum = 0;
+  for (auto& conn : connections_) {
```

Comment:
Nitpicking: const auto&

Reply:
that would make me think, that the `Connection` is `const` but only the `std::shared_ptr<...>` would be
if it is clear for everybody else I can make the change

Reply:
Fair point, not sure which option is better, so feel free to choose. You can also mark this thread resolved.